### ISSUE TITLE
studio(a11y): sign in button not accessible via keyboard only on settings page

### ISSUE THREAD

**Issue by donroyco** [deckgo/deckdeckgo#673]
### Affected part of DeckDeckGo
- [x] Studio
- [ ] Remote control
- [ ] Documentation
- [ ] Demo
- [ ] Starter kit
- [ ] Poll
- [ ] Web Components
- [ ] Templates

### Reproduction

Steps to reproduce:
1. Go to https://deckdeckgo.com/settings 
2. Try to navigate through the page with keyboard only 
 
### Expected Behavior

I expect to reach each element on the page that has a function.

### Actual Behavior

'Sign in' is not reachable with keyboard navigation only.

![Mar-21-2020 10-50-59](https://user-images.githubusercontent.com/1763537/77224016-ec60fb80-6b61-11ea-97fb-1c6035029f80.gif)

_Reproduction of keyboard only navigation. It goes from **logo** to **switch to dark theme** to **URL bar** and skips *Sign in* button._

### Environment

- Browser(s): Chrome, latest version
- Operating System (e.g. Windows, macOS, Ubuntu): macOS

**Comment by peterpeterparker** [comment]
Again, well spotted 😉

There is the same problem in the `dashboard` route too if not logged in.

I think the problem happens when I used `<ion-router-link/>` with `click` event only. I guess at first I used them with navigation and than later to replace with some javascript to navigate, that's why it remained like this.

To solve the issue, my guess, the `<ion-router-link/>` should be modified to `<button/>`.

**Comment by donroyco** [comment]
I am trying to fix this locally, but what should I add as a `class` to get the same look as before, but treating it as a button?

**Comment by peterpeterparker** [comment]
First of all, @boradakash contacted me to inform me that he was also interested to have a look at this question. We exchanged about some questions but don't know if he already began to work on it? I hope you are not both on it...

About style, I'm not sure there is y a `class` for it. Can you create one?

I use the `type="button"` when there is a form and the button is not submitting it but to me you are the a11y export, therefore I follow your advice and judgement.

**Comment by boradakash** [comment]
Hi @peterpeterparker , I didn't started it yet using <button>.
@donroyco can push the changes.

**Comment by peterpeterparker** [comment]
Cool thx for the quick feedback @boradakash 

And thank you both for your help, you are awesome!

**Comment by peterpeterparker** [comment]
Thank you @donroyco for pushing forward the accessibility of DeckDeckGo 🙏
